Source/Scene/Expression.js

```diff
@@ -144,6 +158,18 @@ define([
                 node.evaluate = node._evaluateEquals;
             } else if (node._value === '!==') {
                 node.evaluate = node._evaluateNotEquals;
+            } else if (node._value === '<') {
```

Comment:
Note that all the operators are assuming `number`/`number` comparisons.

* We need to do type checking to make sure, for example, that `4 > 'cat'` is a syntax error.  Please carefully think about our type system and make it reasonably strict, e.g., no unexpected implicit conversions like `true > color('red')`
* We also need to implement operators for `Color` types, e.g., component-wise operators.  If we don't want to do those functions to `Color` (totally cool with me to keep Cesium's API surface area small) then we can just add them a local functions in this file.